src/include/iscsi-lio-server/TargetData.rb

```diff
@@ -0,0 +1,594 @@
+class Backstores
+  def initialize()
+    @backstore_path = nil
+    @re_backstore_path = Regexp.new(/\/[\w\/\.]+\s/)
+    @backstores_list = Array.new
```

Comment:
As RuboCop will tell you, `[]` is just as good as `Array.new` and shorter to type.